### PR TITLE
BLB Lumra, Bellow of the Woods

### PR DESCRIPTION
**Pull request by Ampersandnz**
https://scryfall.com/card/blb/183/lumra-bellow-of-the-woods

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/lumra_bellow_of_the_woods.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Lumra, Bellow of the Woods
+ManaCost:4 G G
+Types:Legendary Creature Elemental Bear
+PT:*/*
+K:Vigilance
+K:Reach
+S:Mode$ Continuous | EffectZone$ All | CharacteristicDefining$ True | SetPower$ X | SetToughness$ X | Description$ CARDNAME's power and toughness are each equal to the number of lands you control.
+SVar:X:Count$Valid Land.YouCtrl
+SVar:BuffedBy:Land
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When CARDNAME enters the battlefield, mill four cards. Then return all land cards from your graveyard to the battlefield tapped.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lumra_bellow_of_the_woods.txt:10]: TriggerDescription doesn't match fully

Comment (Ampersandnz) [reply forge-gui/res/cardsfolder/upcoming/lumra_bellow_of_the_woods.txt]: Cheers, tweaked `enters the battlefield` -> `enters`

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/lumra_bellow_of_the_woods.txt]: CARDNAME -> NICKNAME

Comment (Ampersandnz) [reply forge-gui/res/cardsfolder/upcoming/lumra_bellow_of_the_woods.txt]: Ahh understood

**Review by tool4ever — APPROVED** [review]
thanks

**Pull request merged**